jaxb-ri/jxc/src/main/java/com/sun/tools/jxc/gen/config/NGCCRuntime.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2018, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
this should be changed to `1997, 2020` instead

Reply:
Done.